Add a procedure for creating a redirect that passes CI checks (#1893)
elastic/docs-builder: docs/contribute/redirects.md

   - It is at the same location as the `docset.yml` file. Depending on the repository, this is generally at the root or within the `/docs` folder of the repository.

   :::{note}
   It is possible that there is no `redirects.yml` file yet in some repositories. If that's the case, create it next to the repo's `docset.yml` file. Refer to [](#file-location).
   :::

2. Edit the file. Refer to [](#syntax) to get details on the expected syntax.
   - All paths must be relative to the `redirects.yml` file.
   - You may need to adjust the syntax based on how you'd like to treat anchors within redirected files. Find examples in [](#syntax).

3. Fix all existing links to the moved or removed file within the repository where you're adding the redirect, by updating them to the new correct target, or by removing them if necessary. This is a best practice to keep our content healthy, and mandatory for your PR to pass CI checks.

4. Create a PR with all of the changes made through the previous steps. CI checks run to validate the newly added redirect and the docs build. 

> In case of validation errors related the redirect, double check that your changes comply with all steps of this procedure. If the errors persist, reach out to the docs-engineering team.

Reviewer: does this render as intended without a line break? 

consider adding info on how to reach out to docs eng. also, do we want them to reach out to docs eng, or would a writer doing the initial triage be useful?

> 4. Create a PR with all of the changes made through the previous steps. CI checks run to validate the newly added redirect and the docs build. 
> In case of validation errors related the redirect, double check that your changes comply with all steps of this procedure. If the errors persist, reach out to the docs-engineering team.

Author: ```suggestion
4. Create a PR with all of the changes made through the previous steps. 

CI checks run to validate the newly added redirect and the docs build. 
In case of validation errors related the redirect, double check that your changes comply with all steps of this procedure. If the errors persist, ask @elastic/docs-engineering for assistance in your PR.
```

For elastic.co/guide redirects, open a [web team request](http://ela.st/web-request).

## Add a redirect

To successfully implement a redirect:

1. Locate and open the `redirects.yml` file to edit.
   - Each docs repository powered by Docs V3 can have its own `redirects.yml` file. You must only edit the one in the repository that you're moving or removing files from.

>    - It is at the same location as the `docset.yml` file. Depending on the repository, this is generally at the root or within the `/docs` folder of the repository.

Reviewer 2: very minor suggestion

I'd reorder this to put the info in the second bullet first, followed by `Each docs repository powered by Docs V3 can have its own redirects.yml file. .....` perhaps in a TIP

   :::{note}

>    It is possible that there is no `redirects.yml` file yet in some repositories. If that's the case, create it next to the repo's `docset.yml` file. Refer to [](#file-location).

Reviewer 2: ```suggestion
   Some repositories might not yet have a `redirects.yml` file. In this case, create one next to the repo's `docset.yml` file. Refer to [](#file-location).
```

   :::

2. Edit the file. Refer to [](#syntax) to get details on the expected syntax.
   - All paths must be relative to the `redirects.yml` file.
   - You may need to adjust the syntax based on how you'd like to treat anchors within redirected files. Find examples in [](#syntax).

3. Fix all existing links to the moved or removed file within the repository where you're adding the redirect, by updating them to the new correct target, or by removing them if necessary. This is a best practice to keep our content healthy, and mandatory for your PR to pass CI checks.

4. Create a PR with all of the changes made through the previous steps. 

CI checks run to validate the newly added redirect and the docs build. 

> In case of validation errors related the redirect, double check that your changes comply with all steps of this procedure. If the errors persist, ask @elastic/docs-engineering for assistance in your PR.

Author: ```suggestion
In case of validation errors related the redirect, double check that your changes comply with all steps of this procedure. If the errors persist after a writer reviewed the PR, ask @elastic/docs-engineering for assistance.
```

> In case of validation errors related the redirect, double check that your changes comply with all steps of this procedure. If the errors persist after a writer reviewed the PR, ask @elastic/docs-engineering for assistance.

Reviewer 2: ```suggestion
If you get validation errors about the redirect, double check that your changes follow all the steps in this procedure. If the errors persist after a writer reviewed the PR, ask @elastic/docs-engineering for assistance.
```
